generators/add-action/asset-compute/templates/README.md

```diff
@@ -0,0 +1,79 @@
+# Worker Template
+
+This is a very simple example of a custom Asset Compute worker. It generates a rendition by simply copying the source file.
+
+## Code
+
+The key elements are
+- [worker.js](worker.js) - nodejs action with the worker logic
+- [manifest.yml](manifest.yml) - describes the Runtime action that gets deployed using `aio`
+- [tests/](tests) - worker test cases
+
+In the [manifest.yml](manifest.yml) the single action is set to be a web action and that it requires standard Adobe ID authentication & authorization:
+
+```
+packages:
+  __APP_PACKAGE__:
+    actions:
+      worker:
+        function: 'worker.js'
+        runtime: 'nodejs:10'
+        web: true
+        annotations:
+          require-adobe-auth: true
+```
+
+## Install
+
+Requirements:
+
+* [aio cli](https://github.com/adobe/aio-cli)
+* [asset-compute plugin for aio](https://github.com/adobe/aio-cli-plugin-asset-compute) 
+* Adobe IO Runtime namespace where you want to install the worker into
+* `.env` file in the root directory with the credentials of the Runtime namespace, and these variables:
+  - `AIO_RUNTIME_NAMESPACE` for the namespace name (`NAMESPACE` from `.wskprops`)
+  - `AIO_RUNTIME_AUTH` for the namespace name (`AUTH` from `.wskprops`)
```

Comment:
should we add a note about the devtool credentials here?

Reply:
Please do so.

Reply:
Ok I will do that later

Reply:
Done. Take a look when you have a chance

Reply:
It was tough because our worker documentation guide is not publicly available yet so I had to just repeat a lot of information that is in the guide. When we publish our docs, we can remove some of the repeated info in here